File: app/src/main/java/com/stavro_xhardha/fcbarcelonashqip/MatchFragmentTabAdapter.java
package com.stavro_xhardha.fcbarcelonashqip;


import android.content.Context;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

/**
 * Created by stavro_xhardha on 22/04/2018.
 */

public class MatchFragmentTabAdapter extends FragmentPagerAdapter {
    private Context mContext;

    public MatchFragmentTabAdapter(Context context, FragmentManager fm) {
        super(fm);
        mContext = context;
    }

    @Override
    public Fragment getItem(int position) {
        if (position == 0) {
            return new ScheduledMatchesFragment();
        } else {
            return new MatchHistoryFragment();
        }
    }

    @Override
    public int getCount() {
        return 2;
    }

    @Override
    public CharSequence getPageTitle(int position) {
        switch (position) {
            case 0:
                return mContext.getString(R.string.scheduled_match);
            case 1:
                return mContext.getString(R.string.history_match);
            default:
                return null;
        }
    }
}

File: app/src/main/java/com/stavro_xhardha/fcbarcelonashqip/MatchFragmentTabAdapter.kt
package com.stavro_xhardha.fcbarcelonashqip


import android.content.Context
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

/**
 * Created by stavro_xhardha on 22/04/2018.
 */

class MatchFragmentTabAdapter(private val mContext: Context, fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return if (position == 0) {
            ScheduledMatchesFragment()
        } else {
            MatchHistoryFragment()
        }
    }

    override fun getCount(): Int {
        return 2
    }

    override fun getPageTitle(position: Int): CharSequence? {
        when (position) {
            0 -> return mContext.getString(R.string.scheduled_match)
            1 -> return mContext.getString(R.string.history_match)
            else -> return null
        }
    }
}
